tests/test_unit/test_kinematics.py

```diff
@@ -182,3 +184,120 @@ def test_approximate_derivative_with_invalid_order(order):
     expected_exception = ValueError if isinstance(order, int) else TypeError
     with pytest.raises(expected_exception):
         kinematics._compute_approximate_time_derivative(data, order=order)
+
+
+class TestNavigation:
+    """Test suite for navigation-related functions in the kinematics module."""
+
+    @pytest.fixture
+    def mock_dataarray(self):
+        """Return a mock DataArray containing four known head orientations."""
```

Comment:
I think it would be helpful so that we can reuse this fixture easily to describe a bit more in the docstring.

For example, something like "a position data array, for an individual with 3 keypoints (left ear, right ear and nose) tracked for 4 frames, in x-y space".

Maybe also renaming it more clearly, as `valid_data_array_for_head_vector`